compiler/passes/docs.cpp

```diff
@@ -240,7 +240,9 @@ void createDocsFileFolders(std::string filename) {
     dirCutoff += total;
     std::string shorter = filename.substr(dirCutoff+1);
     std::string otherHalf = filename.substr(0, dirCutoff);
-    mkdir(otherHalf.c_str(), S_IWUSR|S_IRUSR|S_IXUSR);
+    if (otherHalf.length() > 0) {
+      makeDir(otherHalf.c_str());
+    }
```

Comment:
This changes the permissions of the directory.  It used to be rwx------ (or read, write, execute permissions for the user only), but now it is rwxrwxrwx (or read, write, execute permissions for user, group, and other)